src/protocols/vnc/vnc.c

```diff
@@ -49,11 +49,19 @@
 #include <guacamole/wol.h>
 #include <rfb/rfbclient.h>
 #include <rfb/rfbproto.h>
+#ifdef LIBVNCSERVER_WITH_CLIENT_GCRYPT
+#include <asm-generic/errno.h>
```

Comment:
Why this header?

Reply:
The macro for the GCrypt pthread implementation requires it. If you remove it, you get this build error:

```
Making all in src/protocols/vnc
make[2]: Entering directory `/guac/working/guacamole-server/src/protocols/vnc'
  CC       libguac_client_vnc_la-vnc.lo
In file included from vnc.c:53:0:
vnc.c: In function 'gcry_pthread_mutex_init':
vnc.c:61:1: error: 'ENOMEM' undeclared (first use in this function)
 GCRY_THREAD_OPTION_PTHREAD_IMPL;
 ^
vnc.c:61:1: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [libguac_client_vnc_la-vnc.lo] Error 1
make[2]: Leaving directory `/guac/working/guacamole-server/src/protocols/vnc'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/guac/working/guacamole-server'
make: *** [all] Error 2
```

Reply:
I believe `asm-generic/errno.h` is the internal, platform-specific header, and this should just be `errno.h`.

Reply:
Aha, makes sense. will fix momentarily.